doc/source/scalability/innodb_io.rst

```diff
@@ -31,13 +31,32 @@ System Variables
    :default: True
    :range: True/False
 
-This variable is used to control the ability of the user to set the value of the global |MySQL| variable ``innodb_flush_log_at_trx_commit``.
+This variable enables or disables the effect of the per-session value of 
+the `innodb_flush_log_at_trx_commit` variable. 
 
-If ``innodb_use_global_flush_log_at_trx_commit=0`` (False), the client can set the global |MySQL| variable, using: ::
+The following are the available values:
 
-  SET innodb_use_global_flush_log_at_trx_commit=N
+    * ``1`` - Evaluated as ``TRUE`` and is the default value
```

Comment:
It's not evaluated as - this is a boolean variable, the possible values are true and false. (which is represented as 0/1, but that's not really important)

I don't think this and the next line adds any information - the possible values and the default is already mentioned a few paragraphs above.

Reply:
Removed.